adapters/relevantdigital/relevantdigital.go

```diff
@@ -0,0 +1,265 @@
+	return macros.ResolveMacros(a.endpoint, endpointParams)
+}
+
+func (adapter *adapter) buildAdapterRequest(prebidBidRequest *openrtb2.BidRequest, params []*openrtb_ext.ExtRelevantDigital) (*adapters.RequestData, error) {
+	newBidRequest, err := adapter.createBidRequest(prebidBidRequest, params)
+
+	if err != nil {
+		return nil, err
+	}
+
+	reqJSON, err := json.Marshal(newBidRequest)
+	if err != nil {
+		return nil, err
+	}
+
+	url, err := adapter.buildEndpointURL(params[0])
+	if err != nil {
+		return nil, err
+	}
+
+	return &adapters.RequestData{
+		Method:  "POST",
+		Uri:     url,
+		Body:    reqJSON,
+		Headers: getHeaders(prebidBidRequest),
+	}, nil
+}
+
+func (adapter *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	impParams, errs := adapter.getImpressionsInfo(request.Imp)
+	if len(errs) > 0 {
+		return nil, errs
+	}
+
+	bidRequest, err := adapter.buildAdapterRequest(request, impParams)
+	if err != nil {
+		errs = []error{err}
+	}
+
+	if bidRequest != nil {
+		return []*adapters.RequestData{bidRequest}, errs
+	}
+	return nil, errs
+}
+
+func (adapter *adapter) getImpressionsInfo(imps []openrtb2.Imp) (resImps []*openrtb_ext.ExtRelevantDigital, errors []error) {
+	for _, imp := range imps {
+		impExt, err := adapter.getImpressionExt(&imp)
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+		resImps = append(resImps, impExt)
+	}
+	return
+}
+
+func getHeaders(request *openrtb2.BidRequest) http.Header {
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+	headers.Add("Accept", "application/json")
+	headers.Add("X-Openrtb-Version", "2.5")
+
+	if request.Device != nil {
+		if len(request.Device.UA) > 0 {
+			headers.Add("User-Agent", request.Device.UA)
+		}
+		if len(request.Device.IPv6) > 0 {
+			headers.Add("X-Forwarded-For", request.Device.IPv6)
+		}
+		if len(request.Device.IP) > 0 {
+			headers.Add("X-Forwarded-For", request.Device.IP)
+		}
+	}
+	return headers
+}
+
+func getMediaTypeForBid(bid openrtb2.Bid) (openrtb_ext.BidType, error) {
+	switch bid.MType {
+	case openrtb2.MarkupBanner:
+		return openrtb_ext.BidTypeBanner, nil
+	case openrtb2.MarkupVideo:
+		return openrtb_ext.BidTypeVideo, nil
+	case openrtb2.MarkupAudio:
+		return openrtb_ext.BidTypeAudio, nil
+	case openrtb2.MarkupNative:
+		return openrtb_ext.BidTypeNative, nil
+	default:
+		return "", fmt.Errorf("unable to fetch mediaType in multi-format: %s", bid.ImpID)
+	}
+}
+
+func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.RequestData, responseData *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+	if responseData.StatusCode != http.StatusOK {
+		return nil, []error{&errortypes.BadInput{
+			Message: fmt.Sprintf("Unexpected status code: %d. Bad request from publisher. Run with request.debug = 1 for more info.", responseData.StatusCode),
+		}}
+	}
```

Comment:
Do you wanna error out even if your bidder returns 302?
If not then you could use the following built-in functions - 
```
if adapters.IsResponseStatusCodeNoContent(response) {
    return nil, nil
}

if err := adapters.CheckResponseStatusCodeForErrors(response); err != nil {
    return nil, []error{err}
}
	```